server/models/User.js

```diff
@@ -2,12 +2,20 @@ const mongoose = require('mongoose');
 
 const { Schema } = mongoose;
 
-// TODO: This is a placeholder. Update after database schema design is complete.
 const userSchema = new Schema({
   name: {
     type: String,
     required: true,
   },
+  accessToken: {
+    type: String,
+  },
+  email: {
+    type: String,
+  },
+  id: {
+    type: String,
+  },
```

Comment:
will there be any cases when any of the other fields will be empty? and if not does it make sense to just make all these fields required?